lblod/frontend-gelinkt-notuleren
# GN-4266: Reference published decisions

### Overview

Bumps "plugins" to enable published decisions insert

<img width="1200" alt="CleanShot 2023-09-19 at 12 27 30@2x" src="https://github.com/lblod/frontend-gelinkt-notuleren/assets/769698/b3929af7-3c4a-4a60-8b2f-fccc3692573c">


#### Connected issues and PRs:

https://binnenland.atlassian.net/browse/GN-4266


### Setup

1. Checkout
2. `ember s --proxy https://dev.gelinkt-notuleren.lblod.info`

### How to test/reproduce

Check whether it is possible to insert reference for a "Besluit". `decisionsEndpoint` might have to be changed to https://publicatie.gelinkt-notuleren.lblod.info/sparql as the production does not have `/sparql` yet.

### Challenges/uncertainties

Production does not have `/sparql` yet


### Checks PR readiness
